.changeset/cool-spies-return.md

```diff
@@ -0,0 +1,5 @@
+---
+"@equinor/mad-navigation": patch
+---
+
+Fix broken link
```

Comment:
Probably don't need a changeset for this one

Reply:
Or maybe the npm registry wont update the description unless we version bump? @HaakonSvane

Reply:
Yeah it probably won't see this package as publishable unless we do I think